content/committees/specification/tckprocess/index.md

```diff
@@ -216,6 +220,10 @@ update and release of an official distribution of the TCK including the
 new exclude list. The associated `challenge` issue MUST be closed with
 an `accepted` label to indicate it has been resolved.
 
+If the TCK challenge recommends a workaround of adding a specific Jakarta deployment 
```

Comment:
This sounds a little backwards.  The TCK challenge can suggest anything they want.  But, the real driver here is the owner of the TCK.  So, if the TCK owner determines that the challenge is valid and they document a workaround of using a specific Jakarta deployment descriptor, then those tests would not have to be excluded.  We need the TCK owner to determine what are valid workarounds.

Reply:
Okay, if we loosen up what are valid workarounds that is more open ended but works equally as long as the workaround is not a `test code` change.

Reply:
We could rephrase as:
`The specification project may approve workarounds for an accepted TCK challenge so that the tests do not need to be excluded.`  

This would also avoid the need for a new TCK release.

Reply:
@kwsutter I also updated the wording for this.